test/csuite/timestamp_abort/main.c

```diff
@@ -750,7 +750,17 @@ thread_backup_run(void *arg)
      * Find the last successful backup.
      */
     if (td->workload_iteration > 1) {
-        testutil_check(session->open_cursor(session, "backup:query_id", NULL, NULL, &cursor));
+        ret = session->open_cursor(session, "backup:query_id", NULL, NULL, &cursor);
+        /*
+         * If the query indicates no previous backup exists, then we only want to create a full
+         * backup this iteration.
+         */
+        if (ret != 0) {
```

Comment:
This could be written more simply as (without needing to be wrapped within `if (ret != 0)`):
```
            if (ret == EINVAL)
                goto create;
            testutil_check(ret);
```
But feel free to keep it the way you wrote it initially if it's clearer! 😄

Reply:
That is much better. I'll fix that (and I think there's another place to clean up too)